src/styles/theme.scss

```diff
@@ -13,6 +13,7 @@
   --spacing-medium: 1rem;
   --spacing-medium2: 1.25rem;
   --spacing-large: 1.1875rem;
+  --spacing-xlarge: 1.5rem;
```

Comment:
As discussed offline, we should use pixels rather than rem for new features/PRs.